patches/minecraft/net/minecraft/world/gen/ChunkGeneratorOverworld.java.patch

```diff
@@ -100,3 +109,39 @@
          BlockFalling.field_149832_M = false;
      }
  
+@@ -475,7 +508,8 @@
+     public List<Biome.SpawnListEntry> func_177458_a(EnumCreatureType p_177458_1_, BlockPos p_177458_2_)
+     {
+         Biome biome = this.field_185995_n.func_180494_b(p_177458_2_);
+-
++        net.minecraftforge.event.terraingen.ChunkGeneratorEvent.GetPossibleCreatures event=new net.minecraftforge.event.terraingen.ChunkGeneratorEvent.GetPossibleCreatures(this, field_185995_n, p_177458_1_, p_177458_2_);
++        if(net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(event)) return event.getPossibleCreatures();
```

Comment:
Don't remove the blank line.
Also, you are adding whole lines here, format the code properly (missing spaces around the `=` for example).